README.md

```diff
@@ -0,0 +1,3 @@
+# #! book
+
+A FOSS hackers guide to the CLI, privacy, security, self hosting, and the internet.
```

Comment:
technically not "FOSS" as it's not software

Reply:
also not yet licensed so i'm not sure how "free" it is

Reply:
That's fair. I just created it from the repo description. I'd like to have a ToC on the README eventually.

Reply:
I read "FOSS hacker" as one thing.

Reply:
That's fair. I guess I didn't put that together mentally.